### ISSUE TITLE
google apps mail & leaves not working with ICE HRM

### ISSUE THREAD

**Issue by gamonoid** [gamonoid/issues#31]
**[Original report](https://bitbucket.org/thilina/icehrm-opensource/issue/31) by Anonymous.**

----------------------------------------

I've installed ICE HRM v4.0 locally and have been testing it.  A couple of problems I've faced:

1-	with e-mail setup, knowing that I use Google apps email.  I put in the values needed but with no luck
2-	when an employee submits a leaves request, it doesn't give a success message.  Yet if you try to submit again, it will tell you leaves are overlapping.  The first leave would show in pending state, yet it also won’t show at the employee’s supervisor’s list for subordinates.  The system keeps saying “processing” and stays there forever

Kindly help and advise on what to do, knowing that your online instance of the app is functioning fine for point 2.

**Comment by gamonoid** [comment]
**Original comment by Thilina Hasantha (Bitbucket: [thilina](https://bitbucket.org/thilina), GitHub: [thilina](https://github.com/thilina)).**

----------------------------------------

1) Can you refer to http://icehrm-blog.gamonoid.com/configuring-email-settings.

2) There is a known bug in v4.2 which prevent success message when applying a leave from appearing when email sending fail. I'll fix this in next version. So for now the best way to fix your email settings

**Comment by gamonoid** [comment]
**Original comment by Majdi (Bitbucket: [malhasan](https://bitbucket.org/malhasan), GitHub: Unknown).**

----------------------------------------

Thanks Thilina... i actually thought that the leave issue is related to the e-mail sending problem.  The thing is that i've set email settings properly and exactly per your recommendation way before i submitted this request.  I'm using google's apps email and attached a screenshot of the settings.  Thanks for helping.![ICE-HRM email settings.jpg](https://bitbucket.org/repo/x9p9dB/images/165859292-ICE-HRM%20email%20settings.jpg)

**Comment by gamonoid** [comment]
**Original comment by Thilina Hasantha (Bitbucket: [thilina](https://bitbucket.org/thilina), GitHub: [thilina](https://github.com/thilina)).**

----------------------------------------

Did you try using a normal gmail account? Also check whether your server allows out going  connections through 465

**Comment by gamonoid** [comment]
**Original comment by Majdi (Bitbucket: [malhasan](https://bitbucket.org/malhasan), GitHub: Unknown).**

----------------------------------------

Yes sir I did try with my personal gmail account but with no luck.  As far as the server settings go, I'd have to check in the morning, but I believe (not sure) that we have other apps on the server and we don't have any issues with email on them.

Also, I tried to sign up for the cloud hosted ice hrm and it gives me a success message but I don't get a subscription/confirmation email. I tried that twice with two different emails but with no luck.

Regards,

>
